rustls/src/conn.rs

```diff
@@ -981,6 +982,25 @@ impl<Data> ConnectionCore<Data> {
             Err(e) => Err(e.clone()),
         }
     }
+
+    fn refresh_traffic_keys(&mut self) -> Result<(), Error> {
```

Comment:
ordering nit: I think since `maybe_refresh_traffic_keys` calls `refresh_traffic_keys` their order is inverted.